analysis_options.yaml

```diff
@@ -18,6 +18,7 @@ analyzer:
   exclude: # DIFFERENT FROM FLUTTER/FLUTTER
     # Ignore generated files
     - '**/*.mocks.dart' # Mockito @GenerateMocks
+    - 'packages/go_router_builder/example/*' # TODO(skogsfrae): remove in https://github.com/flutter/packages/pull/2679
```

Comment:
We should not be turning off analysis for an entire directory. What problem is this intended to solve?

Reply:
> then I faced another problem for updating a file of a library without increasing its version, so I had to revert it.

The second failure you linked to is only for not having a CHANGELOG entry, not lack of version update, and as the error message says that's trivially overridable when it makes sense to do so (as in this case). Local ignores and an override for the changelog would be much better than changing the root analysis configuration to opt everything out.